src/sagemaker/config/config_schema.py

```diff
@@ -768,6 +879,13 @@ def _simple_path(*args: str):
                     TYPE: OBJECT,
```

Comment:
can leave a quick comment about why [AdditionalInferenceSpecifications] is not supported